google/cloud/forseti/services/utils.py

```diff
@@ -258,3 +259,20 @@ def get_resource_id_from_type_name(type_name):
     if '/' in type_name:
         return type_name.split('/')[-1]
     return type_name
+
+
+def opencensus_enabled():
```

Comment:
This method's naming would be more accurate as `is_opencensus_installed()`, because that's what it's checking.